src/styles/LessonSide.scss

```diff
@@ -9,18 +9,76 @@
 
 #lesson-side-container {
   background-color: colors.$primary-green;
+  display: flex;
+  flex-direction: column;
   height: 100%;
-  padding: 20px;
+  justify-content: space-between;
+  overflow-y: auto;
   width: 40%;
 }
 
+#turtle-logo {
+  color: colors.$text-white;
+  padding: 5%;
+  //align-items: flex-start;
+  text-align: left;
+}
+
+#turtle-logo-text {
+  display: inline-block;
+  font-family: 'PT Sans', sans-serif;
+  font-size: 3.5em;
+  padding-left: 20px;
+  vertical-align: middle;
+}
+
+#turtle-image {
+  display: inline-block;
+  vertical-align: middle;
+  //float:left;
+}
+
+#lesson-side-header {
+  color: colors.$text-white;
+  font-family: 'PT Sans', sans-serif;
+  font-size: 1.5em;
+  padding-bottom: 2.5%;
+  padding-left: 5%;
+  text-align: left;
+}
+
+#lesson-side-body {
+  color: colors.$text-white;
+  font-family: 'Quattrocento Sans', sans-serif;
+  font-size: 1em;
+  padding-left: 5%;
+  padding-right: 5%;
+  padding-top: 2.5%;
+  text-align: left;
+}
+
+#encouraging-turtle {
+  align-items: flex-end;
+  display: flex;
+  flex-shrink: 5;
+  max-width: 400px;
+  min-width: 100px;
+  padding: 5%;
+  width: calc(35vw);
+}
+
+#footer-space {
+  height: 8vh;
+}
+
 // LevelSelector
 
 .selector-box {
   align-items: flex-end;
   display: flex;
-  height: 85%;
+  height: auto;
   justify-content: center;
+  padding-bottom: 1vh;
```

Comment:
Try to be consistent with your units. You use % for padding everywhere else except here. Is there a specific reason for that?